frontendUiApi/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/table/BaseGraphTable.java

```diff
@@ -38,6 +38,15 @@ public Component prepareRenderer(TableCellRenderer renderer, int row, int column
     return component;
   }
 
+  /**
+   * This method should be invoked after each graph table model change but BEFORE {@code revalidate} and {@code repaint}
+   * It cause the max column width is reset and can be properly recalculated in {@code prepareRenderer}
```

Comment:
Hmm but how is `prepareRenderer` invoked, in turn? pls add to comment

Reply:
Also:
```suggestion
   * It resets the max column width so that it can be properly recalculated in {@code prepareRenderer}
```

Reply:
Also, wrt. `prepareRenderer`... maybe instead just rework `prepareRenderer` without a need to invoke a separate method like `resetColumnWidth`? This would simplify the logic if possible

Reply:
Ok, reworked.